app/assets/stylesheets/application.scss

```diff
@@ -10,35 +10,24 @@ $govuk-use-legacy-font: false;
 // for guidance
 @import "govuk_publishing_components/govuk_frontend_support";
 @import "govuk_publishing_components/components/back-link";
-@import "govuk_publishing_components/components/breadcrumbs";
-@import "govuk_publishing_components/components/button";
 @import "govuk_publishing_components/components/checkboxes";
 @import "govuk_publishing_components/components/details";
 @import "govuk_publishing_components/components/document-list";
-@import "govuk_publishing_components/components/error-message";
 @import "govuk_publishing_components/components/error-summary";
-@import "govuk_publishing_components/components/feedback";
 @import "govuk_publishing_components/components/govspeak";
-@import "govuk_publishing_components/components/heading";
-@import "govuk_publishing_components/components/hint";
-@import "govuk_publishing_components/components/input";
 @import "govuk_publishing_components/components/inset-text";
 @import "govuk_publishing_components/components/inverse-header";
-@import "govuk_publishing_components/components/label";
 @import "govuk_publishing_components/components/metadata";
```

Comment:
Side point (slightly unrelated to this PR but might as well get it in) the metadata component has a JS file that isn't included in this app and it should be.